lib/msal-node/src/packageMetadata.ts

```diff
@@ -1,3 +1,4 @@
 /* eslint-disable header/header */
 export const name = "@azure/msal-node";
+/** @public */
 export const version = "2.15.0";
```

Comment:
Am I allowed to edit this file? Is this auto-generated?

Reply:
Yes, ideally supposed to be updated every release and I see this is up-to-date in browser. We will need to do the same for node.